public/js/area-structure-modal.js

```diff
@@ -28,10 +28,10 @@ apos.define('area-structure-modal', {
         })
       })
       self.$el.find('[data-apos-area-structure-edit]').on('click', function() {
-        var selector = '.apos-area-widget[data-apos-widget-id="' + $(this).attr('data-apos-widget-id') + '"]';
+        var selector = '[data-apos-area-edit] .apos-area-widget[data-apos-widget-id="' + $(this).attr('data-apos-widget-id') + '"]:first';
```

Comment:
so we're just skipping out on non-editable areas, yeah? No need to represent them?

Reply:
Technically we could have just a warp button to jump to them, but right now budget is very limited.